Assignment/starter_code/js/index.js

```diff
@@ -0,0 +1,50 @@
+alert("I am an alert box!");
+
+$( document ).ready(function() {
+
+	$('#anchor1').click(function(event){
+    event.preventDefault();
```

Comment:
You set up a click listener which when clicked sets up another click listener?

Essentially you are saying to the program - when user clicks on ```#anchor1```, set up another clicklistener for ```#readmore``` and when user clicks this it shows/hides.  It only works because the click propagates to the parent elements.

Effectively the first part of your code could be 
```
$( document ).ready(function() {

	$('#anchor1').click(function(event){
        event.preventDefault();
		$('#show-this-on-click').slideDown( 'slow' );
		$('.readmore').hide();
		$('.readless').show();
	});

	$('#anchor2').click(function(event){
        event.preventDefault();
		$('.readless').hide();
		$('.readmore').show( 'slow');
		$('#show-this-on-click').slideUp( 'slow' );	    		
	});
});
```